tests/elements-regression/helper.ts

```diff
@@ -78,12 +78,19 @@ export default class ElementRegressionHelper {
 		let label = '';
 		const deviceParams = { mobile: { width: 360, height: 736 }, tablet: { width: 768, height: 787 } };
 
+		// TODO: Fix separately in [ED-17812].
+		if ( 'container_grid' === args.widgetType && args.isPublished && 'mobile' === args.device ) {
+			return;
+		}
+
```

Comment:
Why are we skipping this case?
Please the failing workflow to the task

Reply:
@davseve Updated